src/frontend/src/components/Login/LoggedIn.jsx

```diff
@@ -1,15 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Typography } from '@material-ui/core';
 
 import useSiteMetadata from '../../hooks/use-site-metadata';
 
+const useStyles = makeStyles({
+  button: {
+    float: 'right',
+    margin: '0 0.5rem 0 0.5rem',
+  },
+  link: {
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    color: 'white',
+  },
+});
+
 function LoggedIn(props) {
   const { telescopeUrl } = useSiteMetadata();
   const logoutUrl = `${telescopeUrl}/auth/logout`;
+  const classes = useStyles();
 
   return (
     <div>
-      Welcome {props.email} <a href={logoutUrl}> Logout </a>
+      <Button className={classes.button}>
+        <a href={logoutUrl}> Logout </a>
```

Comment:
I think you just forgot classes.link here

Reply:
thanks